package.json

```diff
@@ -55,6 +56,7 @@
     "underscore": "~1.9.1"
   },
   "devDependencies": {
+    "coveralls": "^3.0.2",
```

Comment:
We try to avoid `^` usage in packages.json dependencies (we have had bad experience with it in the past). Could you change it to `~`, please?